test/govuk_component/tasklist_header_test.rb

```diff
@@ -0,0 +1,35 @@
+require 'govuk_component_test_helper'
+
+class TasklistHeaderTestCase < ComponentTestCase
+  def component_name
+    "tasklist_header"
+  end
+
+  test "renders nothing without passed content" do
+    assert_empty render_component({})
+  end
+
+  test "renders basic component" do
+    render_component(title: "This is my title")
+
+    assert_select ".pub-c-tasklist-header span.pub-c-tasklist-header__title", text: "This is my title"
+  end
+
+  test "renders with a link" do
+    render_component(title: "This is my title", path: "/notalink")
+
+    assert_select ".pub-c-tasklist-header a.pub-c-tasklist-header__title[href=?]", "/notalink", text: "This is my title"
```

Comment:
Can the first bit of this be simplified to 

`assert_select ".pub-c-tasklist-header a.pub-c-tasklist-header__title[href='/notalink']"`

(same applies to other tests using this kind of assertion)

Reply:
I didn't know you could do that, thanks, that's much simpler.